content/docs/tasks/security/authz-permissive/index.md

```diff
@@ -0,0 +1,231 @@
+    kind: ClusterRbacConfig
+    metadata:
+      name: default
+    spec:
+      mode: 'ON_WITH_INCLUSION'
+      inclusion:
+        namespaces: ["default"]
+      enforcement_mode: PERMISSIVE
+    EOF
+    {{< /text >}}
+
+    Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`), you should
+    see everything works fine, same as in [preparation task](#before-you-begin).
+
+1.  Apply YAML file for the permissive mode metric collection.
+
+    Run the following command:
+
+    {{< text bash >}}
+    $ kubectl apply -f @samples/bookinfo/platform/kube/rbac/rbac-permissive-telemetry.yaml@
+    logentry.config.istio.io/rbacsamplelog created
+    stdio.config.istio.io/rbacsamplehandler created
+    rule.config.istio.io/rabcsamplestdio created
+    {{< /text >}}
+
+1.  Send traffic to the sample application.
+
+    For the Bookinfo sample, visit `http://$GATEWAY_URL/productpage` in your web
+    browser or issue the following command:
+
+    {{< text bash >}}
+    $ curl http://$GATEWAY_URL/productpage
+    {{< /text >}}
+
+    Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`), you should
+    see everything works fine.
+
+1.  Verify the logs stream has been created and check `permissiveResponseCode`.
+
+    In a Kubernetes environment, search through the `istio-telemetry`
+    pods' logs as follows:
+
+    {{< text bash json >}}
+    $ kubectl -n istio-system logs -l istio-mixer-type=telemetry -c mixer | grep \"instance\":\"rbacsamplelog.logentry.istio-system\"
+    {"level":"warn","time":"2018-08-30T21:53:42.059444Z","instance":"rbacsamplelog.logentry.istio-system","destination":"ratings","latency":"9.158879ms","permissiveResponseCode":"denied","permissiveResponsePolicyID":"","responseCode":200,"responseSize":48,"source":"reviews","user":"cluster.local/ns/default/sa/bookinfo-reviews"}
+    {"level":"warn","time":"2018-08-30T21:53:41.037824Z","instance":"rbacsamplelog.logentry.istio-system","destination":"reviews","latency":"1.091670916s","permissiveResponseCode":"denied","permissiveResponsePolicyID":"","responseCode":200,"responseSize":379,"source":"productpage","user":"cluster.local/ns/default/sa/bookinfo-productpage"}
+    {"level":"warn","time":"2018-08-30T21:53:41.019851Z","instance":"rbacsamplelog.logentry.istio-system","destination":"productpage","latency":"1.112521495s","permissiveResponseCode":"denied","permissiveResponsePolicyID":"","responseCode":200,"responseSize":5723,"source":"istio-ingressgateway","user":"cluster.local/ns/istio-system/sa/istio-ingressgateway-service-account"}
+    {{< /text >}}
+
+    In the above telemetry logs,  the `responseCode` is 200 which is what user see now.
+    The `permissiveResponseCode` is `denied` which is what user will see after switching
+    global authorization configuration from `PERMISSIVE` mode to `ENFORCED` mode, which
+    indicates the global authorization configuration will work as expected after rolling
+    to production.
```

Comment:
This text fails to convey to readers what actions they should be taking. To the readers, the log is in front of them and they are seeing the log at that moment, thus the use of the future tense is confusing. The paragraph includes information that is not pertinent to the action the reader needs to take. Remove the unrelated information to make the text easier to follow. You do mention how the log changes later on, there is no need to obfuscate the action with that information at this point.
```suggestion
1. Verify that the the log shows a `responseCode` of `200` and a `permissiveResponseCode` of `denied`.
```